packages/form-js-editor/src/features/properties-panel/entries/NumberEntries.js

```diff
@@ -91,7 +91,7 @@ function NumberArrowStep(props) {
     return value;
   };
 
-  const setValue = (value) => editField(field, [ 'increment' ], value);
+  const setValue = (value) => editField(field, [ 'increment' ], value && Big(value).toString());
```

Comment:
It seems like this is not side-effect free. When typing rather low and starting to type the digits, the `.` got erased rather quickly. 

![Kapture 2023-01-10 at 14 02 52](https://user-images.githubusercontent.com/9433996/211558832-49948623-b454-488c-9f02-1e401c0c9b4f.gif)

Reply:
Very nice catch, there are three ways we can fix this if we want to keep trimming

1) We don't treat `45.` as a valid value, and wait for the user to enter a more complete entry:

4 is entered => 4 is serialized
5 is entered => 45 is serialized
. is entered => nothing happens
5 is entered => 45.5 is serialized

The downside of this is, if someone were to enter `45.` quickly, because the field is debounced, we may have no value at all.

2) We serialize the dot in the schema

This avoids the above mentioned downside, but may be a little weird in terms of schema to store something like `23.`

3) We implement something at the properties panel field level to deal with this exact scenario

Downside here is implementation time, I'm not sure how/if such an implementation would work well.

@christian-konrad @pinussilvestrus maybe you have some input here? Personally I think option 2 is the right call.

Reply:
> We serialize the dot in the schema

As far as I can see`45.`  is _valid_ input data for the numbers itself, as it would simply resolve to `45`. You can also specify this via input data; no error is thrown.

Therefore we would be fine persisting it this way in the schema for the increment, as it would be consistent. If we can avoid trimming for such cases, even better - no magic.